src/styles/block/pruduct-card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), colors should be defined in variables for reusability and maintainability. Consider adding this color to your 'variables.scss' if it's used in multiple places.